battleship/prompting.py

```diff
@@ -423,16 +670,33 @@ def to_chat_format(self):
     "Do not include any other explanation or prose.\n"
 )
 
+PROMPT_SYSTEM_QUESTION_COT = (
+    "You are a game-playing agent. "
+    "Read the game instructions and examples carefully.\n"
+)
+
 # Task description
 PROMPT_TASK_BASE_QUESTION = (
     "You will be given a partially-revealed game board. "
-    "Your task is to ask a single question that will help you gain information about the position of the remaining hidden ships on the board. "
+    "Your task is to ask a single question that will help you gain the most information possible about the position of the remaining hidden ships on the board. "
```

Comment:
I'm not sure we should explicitly prompt the LM to maximize information gain. The task is to ask a question that will help figure out the position of the ships, so we should frame it as such:
```suggestion
    "Your task is to ask a question that will help you determine the positions of the remaining hidden ships on the board. "
```
It's also no longer a single question, right?